src/NpmUtilities.js

```diff
@@ -76,21 +76,21 @@ export default class NpmUtilities {
   }
 
   @logger.logifySync()
-  static addDistTag(packageName, version, tag, registry) {
+  static addDistTag(directory, packageName, version, tag, registry) {
```

Comment:
I was getting odd NPM ERR on the final publish step in my build and realized because our modules are all @scoped/module in the package.json but simply `module` in the directory that tagging would only work if you cd into the module before tagging